components/automate-cli/pkg/verifyserver/utils/fiberutils/os.go

```diff
@@ -9,3 +12,41 @@ func ExecuteShellCommand(cmd string) ([]byte, error) {
 func CheckPath(cmd string) (string, error) {
 	return exec.LookPath(cmd)
 }
+
+type ExecCmdService interface {
+	Command(name string, args []string) ([]byte, error)
+}
+
+type ExecCmdServiceImp struct{}
+
+func NewExecCmdServiceImp() ExecCmdService {
+	return &ExecCmdServiceImp{}
+}
+
+func (s *ExecCmdServiceImp) Command(name string, args []string) ([]byte, error) {
+	return exec.Command(name, args...).Output()
+}
+
+type UserCmdService interface {
```

Comment:
Looks like this is not a service, its an utils most probably.
move this to verifyserver/utils/user/user.go and rename `UserCmdService` to `UserUtil`

Reply:
Done